src/Servers/Connections.Abstractions/src/ConnectionContext.cs

```diff
@@ -12,22 +12,50 @@
 
 namespace Microsoft.AspNetCore.Connections
 {
+    /// <summary>
+    /// Encapsulates all information about an individual connection irrespective of the underlying transport.
```

Comment:
```suggestion
    /// Encapsulates all information about an individual connection.
```

I don't think raising the transport is useful here and "irrespective" is one of those "trying to sound smart" words to me ;). Also, I believe there could be transport-specific data in this class, it's just that the class itself isn't transport-specific. I think it's clearer without that clause.